src/main/scala/amba/axi4/Deinterleaver.scala

```diff
@@ -81,13 +85,15 @@ class AXI4Deinterleaver(maxReadBytes: Int)(implicit p: Parameters) extends LazyM
 
         // Transmit the selected burst to inner
         in.r.valid := locked
-        in.r.bits  := Vec(qs.map(_.deq.bits))(deq_id)
+        val deq_bits = WireInit(VecInit(qs.map(_.deq.bits)))
+        in.r.bits  := deq_bits(deq_id)
         (deq_OH.asBools zip qs) foreach { case (s, q) =>
           q.deq.ready := s && in.r.fire()
         }
 
         // Feed response into matching Q
-        out.r.ready := Vec(qs.map(_.enq.ready))(enq_id)
+        val enq_readys = WireInit(VecInit(qs.map(_.enq.ready)))
+        out.r.ready := enq_readys(enq_id)
```

Comment:
Why did you need the WireInit?

Reply:
you can's just do Vec(Seq) anymore. I think this is what the cihsel3 cookbook says to do now.

Reply:
ok you're right `VecInit` alone is enough